app/web/features/profile/view/UserTextAndLabel.tsx

```diff
@@ -41,6 +46,60 @@ export const LabelsReferencesLastActive = ({ user }: Props) => (
   </>
 );
```

Comment:
lol this file has become a bit of dump over time as no components in here are even called `UserTextAndLabel` anymore... 🙈

Reply:
I did some renaming include the name of this file to `userLabels.tsx`